app/templates/account/billing-info/payment-info.hbs

```diff
@@ -0,0 +1,3 @@
+<div class="ui basic {{if isLoading 'loading' ''}} segment">
+  {{forms/user-payment-info-form data=user}}
```

Comment:
Why pass the user in ? Couldn't you use it directly in the component ?

Reply:
You are right, I will fix this